commands/register.go

```diff
@@ -173,18 +173,18 @@ func messageReaction(s *discordgo.Session, m *discordgo.MessageReactionAdd) {
 							log.Info(getImage.Header.Get("content-type"))
 							log.Info(mediaHTTP.Status)
 						} else {
```

Comment:
Are these `else` necessary? Should the function not return if theres an error or what should happen that means we cant remove the else and indent the logging out by one?